src/main/scala/com/fulcrumgenomics/vcf/AssessPhasing.scala

```diff
@@ -56,17 +56,23 @@ import scala.collection.mutable.ListBuffer
       |The input VCFs are assumed to be single sample: the genotype from the first sample is used.
       |
       |Only bi-allelic heterozygous SNPs are considered.
+      |
+      |The input known phased variants can be subsetted using the known interval list, for example to keep only variants
+      |from high-confidence regions.
+      |
+      |If an interval list is given, only the set of chromosomes specified will be analyzed.  Note that the full
```

Comment:
"If the intervals argument is supplied" would be clearer since the prior line is discussing an interval list for the known variants.